packages/embed/src/factories/create-popup/create-popup.spec.ts

```diff
@@ -1,32 +1,37 @@
 import { createPopup, Popup } from './create-popup'
 
+jest.useFakeTimers()
```

Comment:
what's this?

Reply:
https://jestjs.io/docs/en/timer-mocks

When you use `setTimeout` in your code, it will get executed when you call `jest.runAllTImers()`. You dont have to actually wait.